application/kitura/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ibmcom/kitura-ubuntu:latest
+
+# Define a working directory
+RUN mkdir HelloKitura
+WORKDIR ./HelloKitura
+
+# Install SecretHub
+RUN echo "deb [trusted=yes] https://apt.secrethub.io stable main" > /etc/apt/sources.list.d/secrethub.sources.list && apt-get update
+RUN apt-get install -y secrethub-cli
+
+# Initialize the application as an executable
+RUN swift package init --type executable
+
+# Copy the necessary files
+COPY Package.swift /HelloKitura/Package.swift
+COPY main.swift /HelloKitura/Sources/HelloKitura/main.swift
```

Comment:
I think it would be nice to have these files under their proper paths also in the example code, not just in the docker image.

Reply:
Alright, I believe this applies to all the other PRs, so I'll do it for all of them.